bucket/fork.json

```
@@ -0,0 +1,42 @@
+{
+    "version": "1.37.2",
+    "description": "A fast and friendly git client for Mac and Windows",
+    "homepage": "https://git-fork.com/",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://git-fork.com/license"
+    },
+    "url": "https://fork.dev/update/win/Fork-1.37.2-full.nupkg",
+    "hash": "sha1:abaa753d9cbeeeb157ec9ba7b5865bf7a2979986",
+    "extract_dir": "lib\\net45",
+    "post_install": [
+        "$gitVersion = (Get-Item \"$dir\\PortableGit*\") -Replace '[^\\d]+([\\d.]+).+', '$1'",
```

```suggestion
        "$gitVersion = (Get-Item \"$dir\\PortableGit*\") -Replace '.*-([\\d.]+)-64-bit\\.7z$', '$1'",
```

This is more strict and should also work in installer.